client/my-sites/sidebar/sidebar.jsx

```diff
@@ -326,7 +326,15 @@ module.exports = React.createClass( {
 			linkClass += ' is-paid-plan';
 		}
 
-		let planName = site.plan.product_name_short;
+		let planName = site.plan.product_name_short,
+			labelClass;
+
+		if ( abtest( 'plansUpgradeButton' ) === 'button' && productsValues.isFreePlan( site.plan ) ) {
+			labelClass = 'add-new';
+			planName = 'Upgrade'; // TODO: translate this string if the test is removed
+		} else {
+			labelClass = 'plan-name';
+		}
```

Comment:
I like omitting the `else` in favor of overwriting `labelClass`. However, adding an unnecessary `else if` here makes it a little harder for me to understand. `planName` should be set to 'Trial' regardless of the `abtest` / `isFreePlan` check in the previous condition, and using `else if` makes it seem (to me) like these conditions are related. As a very general practice, I try to avoid `else if` because I find it easier to read two separate conditions.

What do you think of the following?

```js
let planName = site.plan.product_name_short,
	labelClass = 'plan-name';

if ( abtest( 'plansUpgradeButton' ) === 'button' && productsValues.isFreePlan( site.plan ) ) {
	labelClass = 'add-new';
	planName = 'Upgrade'; // TODO: translate this string if the test is removed
}

if ( productsValues.isFreeTrial( site.plan ) ) {
	planName = this.translate( 'Trial' );
}
```